src/app/globals.css

```
@@ -16,6 +16,8 @@
   --font-weight-semibold: 600;
   --font-weight-bold: 700;
 
+  --color-sdp-main-primary: #ccfb55;
+  --color-sdp-main-secondary: #efffc7;
   --color-sdp-main-primary: #ccfb55;
```

Stylelint (17.11.1)

[error] Duplicate custom property "--color-sdp-main-primary" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)